rhel7/templates/csv/packages_installed.csv

```diff
@@ -7,6 +7,7 @@ dconf
 docker
 esc
 firewalld
+gdm
```

Comment:
@yuumasato doesn't this also need to be added to RHEL6? 
```
grep -r package_gdm_install rhel6

rhel6/checks/oval/gconf_gdm_enable_warning_gui_banner.xml:      <extend_definition comment="gdm installed" definition_ref="package_gdm_installed" negate="true" />
rhel6/checks/oval/gconf_gdm_disable_user_list.xml:      <extend_definition comment="gdm installed" definition_ref="package_gdm_installed" negate="true" />
rhel6/checks/oval/gconf_gdm_set_login_banner_text.xml:      <extend_definition comment="gdm installed" definition_ref="package_gdm_installed" negate="true" />
```

Reply:
@redhatrises  RHEL6 CSV already has `gdm`:
https://github.com/OpenSCAP/scap-security-guide/pull/3047/files/b5e215aad523af3dd8c89ec50213eded9c69a5ae#diff-a3a9a514a045c7876712ddfa567f556fL7
Or maybe I didn't understand your question.

Reply:
@yuumasato nope. I didn't check the RHEL6 csv.